src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -29,6 +29,7 @@ namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Operations.Export
 {
     public class ExportJobTaskTests
     {
+        private const string PatientFileName = "Patient.ndjson";
```

Comment:
nit: Can be removed.

Reply:
Updated the code to actually use the constant.